conf/defaults.ini

```diff
@@ -559,6 +559,20 @@ facility =
 # Syslog tag. By default, the process' argv[0] is used.
 tag =
 
+[log.frontend]
```

Comment:
You'll need to add these changes in conf/sample.ini also, since it has to be in sync with conf/defaults.ini.

Reply:
huh, was sure I did that. thanks!
- [ ] update sample.ini